apikeys/go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/confluentinc/ccloud-sdk-go-v2/apikeys
+
+go 1.13
```

Comment:
this is long EOL'd (minimum is go 1.17 now iirc)... funny that we generate this.